Tools/TestWebKitAPI/Tests/WebKitCocoa/WKWebExtensionAPIBookmarks.mm

```diff
@@ -134,6 +134,23 @@ @interface TestBookmarksDelegate : NSObject <WKWebExtensionControllerDelegatePri
     return nil;
 }
 
+static NSMutableDictionary *findBookmarkAndParentArrayInMockTree(NSMutableArray *rootChildren, NSString *bookmarkId)
+{
+    if (!rootChildren)
```

Comment:
Nit: `if (!rootChildren.count)` - we could return early if it's an empty folder as well.